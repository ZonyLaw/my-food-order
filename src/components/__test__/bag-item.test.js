import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom";
import BagItem from "../bag-item/bag-item.component";

afterEach(() => {
  cleanup();
});

test("renders learn react link", () => {
  const bagitemMock = {
    name: "Ginger Fry beef",
    quantity: 2,
    restuarant: "Little Green",
    price: 22.4,
    imageUrl:
      "https://res.cloudinary.com/dvveiqb1p/image/upload/v1675369770/myFoodOrder/Bistros_hgdxhl.jpg",
  };
  const component = renderer.create(<BagItem bagItem={bagitemMock} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
