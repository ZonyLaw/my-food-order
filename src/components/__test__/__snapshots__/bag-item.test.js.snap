// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="dropdown-container"
>
  <h2>
     
  </h2>
  <div
    className="dropdown-content"
  >
    <img
      alt="Ginger Fry beef"
      className="image"
      src="https://res.cloudinary.com/dvveiqb1p/image/upload/v1675369770/myFoodOrder/Bistros_hgdxhl.jpg"
    />
    <span>
      Ginger Fry beef
      , 
      2
       X Â£
      22.4
    </span>
  </div>
</div>
`;
